### PR TITLE
Update transmission.markdown

### PR DESCRIPTION
**Pull request by SeanPM5**
## Proposed change

Rewrote the “Setup” paragraph a bit, the instructions are now generalized enough to apply to both Linux and macOS unlike before. I also removed a sentence that was implying once the app was configured badges would show up in Lovelace. That was not accurate and confusing, it relies on the integration being added to Home Assistant.

Also a bunch of spelling and grammar fixes, typos, capitalization, punctuation, style tweaks etc.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/transmission.markdown [hunk 1]
```diff
@@ -14,27 +14,15 @@ ha_codeowners:
 ha_domain: transmission
 ---
 
-The `transmission` integration allows you to monitor your downloads with [Transmission](https://www.transmissionbt.com/) from within Home Assistant and setup automation based on the information.
-
-## Setup
```
Comment (fabaff) [inline source/_integrations/transmission.markdown:19]: This heading should not be removed. The integration is pretty old and the chances are high links point here.

File: source/_integrations/transmission.markdown [hunk 2]
```diff
@@ -14,27 +14,15 @@ ha_codeowners:
...
-
-<p class='img'>
-  <img src='/images/integrations/transmission/transmission.png' />
-</p>
+The Transmission integration allows you to monitor your [Transmission](https://www.transmissionbt.com/) BitTorrent downloads from within Home Assistant and set up automations based on that information.
 
 ## Configuration
 
-Set up the integration through **Configuration** -> **Integrations** -> **Transmission**. For legacy support old transmission configuration is imported and set up as new integration. Make sure to remove `monitored_condiditions` as they are now automatically added to Home Assistant
+Your Transmission client must first be configured to allow remote access. In your Transmission client navigate to **Preferences** -> **Remote** tab and then click the **Allow remote access** checkbox.
```
Comment (fabaff) [inline source/_integrations/transmission.markdown:21]: This is part of the setup not the Home Assistant configuration.

**Review by fabaff — CHANGES_REQUESTED** [review]

File: source/_integrations/transmission.markdown (hunk 1 above)
Comment (SeanPM5) [reply source/_integrations/transmission.markdown:19]: OK, restored the setup heading and moved the text back to that section. Should be good now I think.

**Review by frenck — APPROVED** [review]
Thanks, @SeanPM5 👍 

Happy Hacktoberfest 🎉

**Pull request merged**